src/dxtbx/model/detector.py

```diff
@@ -774,9 +774,9 @@ def imgCIF_H(cbf_handle, sensor):
         size = tuple(reversed(cbf_handle.get_image_size(0)))
 
         try:
-            underload = find_undefined_value(cbf_handle)
+            undefined_value = find_undefined_value(cbf_handle)
             overload = cbf_handle.get_overload(0)
```

Comment:
This does also beg the question as to whether the `overload` value is the maximum trusted pixel value, or 1 greater than the maximum trusted pixel value?
https://www.iucr.org/__data/iucr/cifdic_html/2/cif_img.dic/Iarray_intensities.overload.html

Reply:
My interpretation of "saturation intensity level for this data array" is that it is the highest value you can get either by properly counting to that value, or there were higher counts than that, but the detector cannot tell you. As you can't tell the difference between those cases it is an untrusted value.

Unfortunately XDS treats "OVERLOAD" differently, as the highest _acceptable_ pixel count. That leads to confusion in interpretation, as many users are likely to supply the XDS value for this field.

Reply:
XDS is outside the scope of this particular bit of code however - all we are concerned about is "how does the imgCIF `overload` value relate to the dxtbx definition of trusted range, i.e. maximum trusted pixel?" In which case, according to your interpretation, we should subtract 1 from this value in order to set the dxtbx trusted range.

Reply:
My point about XDS is that the definition is ambiguous, demonstrated by the fact that other packages treat the same-named concepts differently. I'm not sure the ambiguity is resolved at all within the imgCIF definitions. In that case, subtracting 1 makes sense, because it is definitely more trustworthy than taking the ambiguous boundary value.